projects/ngrx.io/content/guide/component/let.md

```diff
@@ -5,8 +5,27 @@ The `*ngrxLet` directive serves a convenient way of binding observables to a vie
 
 ## Usage
 
-The `*ngrxLet` directive is provided through the `LetModule`.
-To use it, add the `LetModule` to the `imports` of your standalone component or NgModule:
+The `*ngrxLet` directive is a standalone directive.
+To use it, add the `LetDirective` to the `imports` of your standalone component or NgModule. Alternatively, you can use `LetModule`, however, keep in mind that the `LetModule` is deprecated.
+
+Using the standalone `LetDirective`:
+
+```ts
+import { Component } from '@angular/core';
+import { LetDirective } from '@ngrx/component';
+
+@Component({
+  // ... other metadata
+  standalone: true,
+  imports: [
+    // ... other imports
+    LetDirective,
+  ],
+})
+export class MyStandaloneComponent {}
+```
+
+@deprecated Using `LetModule`:
```

Comment:
I'm not Brandon, but this causes to be the problem.
Because of the `@deprecated` the rest of the content is not rendered (don't ask me why 😅).
Let's remove this sentence and the example below to get this working.

Reply:
thank you @timdeschryver